src/main/java/de/hhu/stups/plues/ui/controller/Reports.java

```diff
@@ -150,6 +223,14 @@ public void initialize(final URL location, final ResourceBundle resources) {
     lbSessionAmount.setText(String.valueOf(sessionAmount));
     lbModelVersion.setText(String.valueOf(properties.get("model_version")));
 
+    this.resources = resources.keySet().stream()
+        .filter(s -> s.startsWith("title.") || s.startsWith("column")).collect(Collectors.toList())
+        .stream().collect(Collectors.toMap(o -> o, resources::getString));
+  }
+
+  @FXML
+  public void print() {
+    realReportData.print();
```

Comment:
Handle the result somehow. Open file, open save file dialogue or offer both options once the file has been generated.